sandbox/java/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.hellosign.sandbox</groupId>
+    <artifactId>HelloSignSandbox</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>19</maven.compiler.source>
+        <maven.compiler.target>19</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.hellosign</groupId>
+            <artifactId>hellosign-java-sdk</artifactId>
+            <version>6.0.0-beta*</version>
```

Comment:
I'm getting `[ERROR] 'dependencies.dependency.version' for com.hellosign:hellosign-java-sdk:jar must not contain any of these characters \/:"<>|?* but found * @ line 21, column 22`.  Let's use `LATEST` here. I know there's a warning regarding the reproducibility of the build, but this should be the easiest fix.

Reply:
`LATEST` seems to always want to point to `5.2.3` which is the non-openapi version. I'll set to `^6.0.0-beta`